### PR TITLE
William loy rse storage maintenance

### PR DESCRIPTION
**Pull request by stephamon**
#2095 

added images and edited text

### CODE REVIEW

File: content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+product: Rackspace Email
+product_url: rackspace-email
+---
+
+This article describes how to monitor user storage in the Cloud Office Control Panel as well as best practices for mailbox storage.
+
+### Prerequisites
+
+- **Applies to:** Administrator
+- **Tools required:**  Cloud Office Control Panel access
```
Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: No difficulty on this one?

File: content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+   <img src="{% asset_path rackspace-email/rackspace-email-storage-maintenance-and-best-practices/RSEstorageWebmailSC1.png %}" />
+
+**Note:** Consider the storage shown in [apps.rackspace.com](https://apps.rackspace.com/index.php) to be the authority for your storage available on the server.
+
+### Recommendations for users nearing their storage capacity
+
+Notify users near capacity and counsel them on archiving items to free up mailbox storage. Rackspace Email Archiving ensures that email traffic to and from your domain is archived. For more information, see [Enable email archiving: Cloud Office Control Panel](/how-to/enable-email-archiving-cloud-office-control-panel/).
+
+If a user deletes an excessive number of emails, this change will take longer to process. Allow ample time for that deletion to be reflected.
```
Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: By "this change" are we referring to enabling email archiving or something else?

File: content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+2. Delete the message from the **Trash** folder.   
+
+The message no longer counts toward the mailbox storage. If you need to recover a message that you deleted from trash, see [Recover deleted email in Webmail](/how-to/recover-deleted-email-in-webmail/).
+
+**Warning:** Messages purged from the **Trash** folder can be recovered for up to 14 days after deletion. After 14 days, the message cannot be recovered.
+
+### Mailbox storage best practices
+
+- Enable [Rackspace Email Archiving](/how-to/enable-email-archiving-cloud-office-control-panel/) for all domains.    
+- Rackspace Email storage is maxed at 25GB. Consider upgrading to our [Microsoft Exchange](https://www.rackspace.com/email-hosting/hosted-exchange) or [Office 365](https://www.rackspace.com/office-365) solutions for higher storage limits.
```
Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: Delete **our** and **solutions**. "Considering upgrading to Microsoft Exchange or Office 365 for higher storage limits."

File: content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- Never store over 10,000 items in one folder.
+- You should not exceed more than 3 levels of subfolders.
+
+    Example folder structure:
+
+    - Best practice folder(level 1)
```
Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: We'll have to check the spacing on this after we merge.

File: content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+        - Best practice folder(level 2)
+
+            - Best practice folder(level 3)
+
+                - Not best practice folder
+
+- Consider removing items from your **Spam** or **Trash** folders.
+- If you archive or delete messages on your [local mail client](/how-to/cloud-office-support-terminology/#cloud-office-terminology), log in to [apps.rackspace.com](https://apps.rackspace.com/index.php) to ensure that those changes are reflected in webmail.
```
Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: ...log in to make sure that those changes...

Comment (stephamon) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: Ensure instead of make sure is a Kelly preference

Comment (DonNateR) [inline content/rackspace-email/rackspace-email-storage-maintenance-and-best-practices.md]: You would know better than I would.

**Pull request merged**